selfdrive/car/honda/interface.py

```diff
@@ -40,7 +40,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     CAN = CanBus(ret, fingerprint)
 
     if candidate in HONDA_BOSCH:
-      cfgs = [get_safety_config(car.CarParams.SafetyModel.ford)]
```

Comment:
@sshane You had initially set this to ford, assuming this was just a copy paste from ford/interface with the same cod?